tasks/section1.yml

```diff
@@ -497,7 +497,7 @@
       - rule_1.1.15
 
 - name: "SCORED | 1.1.16 | AUDIT | Ensure nosuid option set on /dev/shm partition"
-  command: /bin/true
+  shell: mount | grep /dev/shm | grep nosuid
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*